### ISSUE TITLE
Accepting filePath in `packageJson`

### ISSUE THREAD

**Issue by thetutlage** [sapegin/mrm-core#28]
All file related API's accepts the argument to the define file name/path.

```js
json('foo.json')
```

If the path is `absolute`, then it simply writes to that absolute path, which silently opens up the possibility to write contents to different directory by passing absolute paths.

```js
json(join(someBasePath, 'foo.json'))
```

However, this is not possible with the `packageJson` file. Do you think, we can accept the 2nd argument as the filePath?

**Comment by sapegin** [comment]
Yeah, sounds like a good feature, feel free to send a pull request ;-)
